tests/load/gcp/deploy.sh

```diff
@@ -0,0 +1,125 @@
+START="gcloud compute instances start --project=$PROJECT --zone=$ZONE"
+SSH="gcloud compute ssh --project $PROJECT --zone $ZONE"
+
+# LMA spec
+NCPU=2
+GBMEM=4
+INSTANCE="n1-ssd-${NCPU}cpu-${GBMEM}gb"  # e.g. n1-ssd-2cpu-8gb
+AVALANCHE_INSTANCE="avalanche-$INSTANCE"  # e.g. avalanche-n1-ssd-2cpu-8gb
+URL_LMA="$INSTANCE.$ZONE.c.$PROJECT.internal"  # e.g. n1-ssd-2cpu-8gb.c.lma-light-load-testing.internal
+URL_AVALANCHE="$AVALANCHE_INSTANCE.$ZONE.c.$PROJECT.internal"  # e.g. avalanche-n1-ssd-2cpu-8gb.c.lma-light-load-testing.internal
+
+start_avalanche () {
+  if [[ $(gcloud compute instances describe "${AVALANCHE_INSTANCE}") ]]; then
+    echo "avalanche instance ${AVALANCHE_INSTANCE} already exists"
+    gcloud compute instances stop "${AVALANCHE_INSTANCE}"
+  else
+    echo "Creating avalanche instance ${AVALANCHE_INSTANCE}"
+    $CREATE "${AVALANCHE_INSTANCE}" --machine-type=e2-standard-4 --create-disk=auto-delete=yes,boot=yes,device-name=disk-"${AVALANCHE_INSTANCE}",image=projects/lma-light-load-testing/global/images/avalanche,mode=rw,size=10,type=projects/lma-light-load-testing/zones/us-central1-a/diskTypes/pd-balanced
+  fi
+
+  echo "Starting avalanche instance..."
+  $START "${AVALANCHE_INSTANCE}"
+  sleep 60
+  echo "Starting avalanche workers..."
+  $SSH "${AVALANCHE_INSTANCE}" -- 'for ((i = 1 ; i <= 20 ; i++)) ; do port=$((9000 + $i)); `nohup avalanche --metric-count=1000 --label-count=10 --series-count=10 --value-interval=30 --series-interval=36000000 --metric-interval=36000000 --port=$port 1>/dev/null 2>&1 &` ; done'
+}
+
+start_lma () {
+  # $1 = number of avalanche targets to scrape
+  NUM_SCRAPE_TARGETS=$1
+
+  # create SSD instance
+  echo "Creating $INSTANCE..."
+  $CREATE $INSTANCE --machine-type=custom-${NCPU}-$(( GBMEM * 1024 )) --create-disk=auto-delete=yes,boot=yes,device-name=disk-${INSTANCE},image=projects/lma-light-load-testing/global/images/juju-hirsute-dns-ingress,mode=rw,size=50,type=projects/lma-light-load-testing/zones/us-central1-a/diskTypes/pd-ssd
+
+  # start instance
+  echo "Starting $INSTANCE..."
+  gcloud compute instances start $INSTANCE --project=$PROJECT --zone=$ZONE
+  sleep 30
+
+  # test comms with avalanche
+  echo "Testing comms with ${URL_AVALANCHE}..."
+  $SSH $INSTANCE --command=<<EOF
+curl http://${URL_AVALANCHE}:9001/metrics | wc -l
+curl http://${URL_AVALANCHE}:9002/metrics | wc -l
+EOF
+
+  # add current user to group
+  echo "Adding $USER to the microk8s group..."
+  $SSH $INSTANCE --command=<<EOF
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+EOF
+
+  SCRAPE_TARGETS="${URL_AVALANCHE}:9001"
+  for ((i = 2 ; i <= NUM_SCRAPE_TARGETS ; i++)) ; do SCRAPE_TARGETS="${SCRAPE_TARGETS},${URL_AVALANCHE}:$((9000 + $i))"; done
+
+  # deploy lma-light bundle
+  echo "Deploying lma-light..."
+  $SSH $INSTANCE --command=<<EOF
```

Comment:
This script should be broken out into its own thing, uploaded with scp then invoked using ssh. It should also specify `set -euo pipefail`